.npmrc

```diff
@@ -0,0 +1 @@
+if-present = true
```

Comment:
why this?

Reply:
No, but the command was running the root's `test` command that navigates the project's tree and executes all the `.test.js` files.

The right command to execute in this case should be `yarn workspaces run test`
 https://classic.yarnpkg.com/en/docs/cli/workspaces#toc-yarn-workspaces-run